command/exclude_test.go

```diff
@@ -0,0 +1,35 @@
+package command
+
+import "testing"
+
+func Test_wildCardToRegexp(t *testing.T) {
+	t.Parallel()
+	tests := []struct {
+		name    string
+		pattern string
+		wanted  string
+	}{
+		{
+			name:    "wildcard to regex : main*",
+			pattern: "main*",
+			wanted:  "^main.*$",
+		},
+		{
+			name:    "wildcard to regex : *.txt",
+			pattern: "*.txt",
+			wanted:  "^.*\\.txt$",
+		},
+		{
+			name:    "wildcard to regex : ?_main*.txt",
+			pattern: "?_main*.txt",
+			wanted:  "^._main.*\\.txt$",
+		},
```

Comment:
No need to add parent test names to sub test names as hey already has them;
https://blog.golang.org/subtests#TOC_5.
> The full name of a subtest or sub-benchmark is a slash-separated list of its name and the names of all of its parents, starting with the top-level.

Reply:
deleted the `wildcard to regex :` in each sub test name.